docs: Fix a few typos

timgates42 opened the pull request:
There are small typos in:
- avr/cores/tiny/wiring.c
- avr/extras/ATtiny_1634.md
- avr/extras/ATtiny_828.md
- avr/extras/ATtiny_x41.md
- avr/libraries/Wire/src/SoftI2CMaster.h

Fixes:
- Should read `charging` rather than `chargeing`.
- Should read `bootloader` rather than `booloader`.
- Should read `approx` rather than `aprox`.
- Should read `things` rather than `thingss`.
- Should read `stronger` rather than `strongler`.
- Should read `communicating` rather than `communicationg`.



Semi-automated pull request generated by
https://github.com/timgates42/meticulous/blob/master/docs/NOTE.md

timgates42 committed:
docs: Fix a few typos

There are small typos in:
- avr/cores/tiny/wiring.c
- avr/extras/ATtiny_1634.md
- avr/extras/ATtiny_828.md
- avr/extras/ATtiny_x41.md
- avr/libraries/Wire/src/SoftI2CMaster.h

Fixes:
- Should read `charging` rather than `chargeing`.
- Should read `bootloader` rather than `booloader`.
- Should read `approx` rather than `aprox`.
- Should read `things` rather than `thingss`.
- Should read `stronger` rather than `strongler`.
- Should read `communicating` rather than `communicationg`.

SpenceKonde commented:
Hell yeah! Unfortunately, master is a kind of a dead branch here. All the big changes are going in the 2.0.0-dev branch. But I'll merge this, and hopefully remember this when I get to 2.0.0 (megaTinyCore and DxCore have swallowed an incredible amount of tyime, I expected 2.0.0 to have been released in july, after 2.4.0 megaTinyCore late may and a DxCore version in june. I'm hoping to release 2.4.0 of ,megaTinyCore in a day or two,. 

Don't suppose you'd care to aim this at my megaTinyCore and DxCore too? 
I keep finding all sorts of spelling mistakes whenever I update the markdown, because all it gets is codespell.

timgates42 commented:
Thanks @SpenceKonde - sure thing I just added support for targeted scans - it was originally just pulling repositories from various "Awesome" lists in GitHub - I've added in https://github.com/timgates42/repository_list/commit/005155816ce39e7d3f62b30665050e0b4fda89b0
